README.md

```diff
@@ -51,6 +51,30 @@ Set of datasets to bootstrap a working dialogue model for human interaction
 - Uses zmq to talk to other toolboxes not in Python, examples of Lua Torch given.
 - Supports hogwild and batch training of models.
 
+## Requirements
```

Comment:
@jaseweston Always Installation section would be first part instead of examples to execute on environment ready machine